### PR TITLE
Add script for inference by using checkpoint

### LINKED ISSUE #731: Add script for inference by using checkpoint
Due to #724

### PR DESCRIPTION
**Pull request by kuke**
Resolve #731

### CODE REVIEW

File: fluid/DeepASR/infer_by_ckpt.py [hunk 1]
```diff
@@ -0,0 +1,160 @@
...
+        type=str,
+        default='data/global_mean_var_search26kHr',
+        help="The path for feature's global mean and variance. "
+        "(default: %(default)s)")
+    parser.add_argument(
+        '--infer_feature_lst',
+        type=str,
+        default='data/infer_feature.lst',
+        help='The feature list path for inference. (default: %(default)s)')
+    parser.add_argument(
```
Comment (pkuyym) [inline fluid/DeepASR/infer_by_ckpt.py:72]: Please notice the alignment.

File: fluid/DeepASR/infer_by_ckpt.py [hunk 2]
```diff
@@ -0,0 +1,160 @@
...
+        '--infer_label_lst',
+        type=str,
+        default='data/infer_label.lst',
+        help='The label list path for inference. (default: %(default)s)')
+    parser.add_argument(
+        '--checkpoint',
+        type=str,
+        default='./checkpoint',
+        help="Checkpoints. Do not save checkpoint if set to ''. (default: %(default)s)"
```
Comment (pkuyym) [inline fluid/DeepASR/infer_by_ckpt.py]: Seems exceeded 80 columns.

File: fluid/DeepASR/infer_by_ckpt.py [hunk 3]
```diff
@@ -0,0 +1,160 @@
...
+def print_arguments(args):
+    print('-----------  Configuration Arguments -----------')
+    for arg, value in sorted(vars(args).iteritems()):
+        print('%s: %s' % (arg, value))
+    print('------------------------------------------------')
+
+
+def infer_from_ckpt(args):
+    """inference by using checkpoint.
+    """
```
Comment (pkuyym) [inline fluid/DeepASR/infer_by_ckpt.py]: """inference by using checkpoint."""

Comment (kuke) [reply fluid/DeepASR/infer_by_ckpt.py]: Done

File: fluid/DeepASR/infer_by_ckpt.py (hunk 2 above)
Comment (kuke) [reply fluid/DeepASR/infer_by_ckpt.py]: Done

File: fluid/DeepASR/infer_by_ckpt.py (hunk 1 above)
Comment (kuke) [reply fluid/DeepASR/infer_by_ckpt.py:72]: Skipped

**Review by pkuyym — APPROVED** [review]
LGTM

**Pull request merged**